website/css/style.css

```diff
@@ -28,10 +35,139 @@
          url('../fonts/roboto-v27-latin-500.ttf') format('truetype'), /* Safari, Android, iOS */
          url('../fonts/roboto-v27-latin-500.svg#Roboto') format('svg'); /* Legacy iOS */
   }
-.container {
-    margin-left: 104px;
+
+  /*+++++++++++++++Header+++++++++++++++*/
+
+.header {
+    background-image: url("../img/castelmezzano.svg");
+    height: 900px;
+    max-width: 1440px
+}
+.header_inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 51px 104px 0;
+}
+.header_logo {
     margin-right: 104px;
-  }
+}
+.nav {
+    display: flex;
+    align-items: center;
+    font-size: 24px;
+    font-weight: 400;
+}
+.nav_link {
+    color: #fff;
+    text-decoration: none;
+    margin-right: 24px;
+}
+
+.nav_link_icon {
+    text-decoration: none;
+    margin-left: 24px;
+}
+.nav :hover {
+    border-bottom: solid 4px #F5BD41;
+}
```

Comment:
I would advise to do more precise styles. like `.nav a:hover` to avoid double bottom line
![image](https://user-images.githubusercontent.com/20189313/127791639-2c5267ea-7df6-4015-91ba-69c03473e882.png)

Reply:
everything else is good